tests/model/test_model.py

```diff
@@ -1312,7 +1312,8 @@ def test_notify_users_enabled(
         assert notify.called == is_notify_called
 
     @pytest.mark.parametrize(
-        "event, expected_times_messages_rerendered, expected_index, topic_view_enabled",
+        "event, expected_times_messages_rerendered, expected_index, topic_view_enabled"
+        ", mentioned_msg_ids",
```

Comment:
I know I simplified a lot of these into single strings; we can discuss the preferred style of whether to break into a multi-line implicitly-combined string like this or a list of strings, but at minimum I would expect a comma to belong with the preceding text?

Reply:
Personally, I would prefer the multiline string approach since that saves a lot of lines compared to the list approach, where each parameter would be shifted to a new line hence increasing the diff. But, the list approach may be a bit more comprehensible than the string version.
I am going with the multiline string approach here, for now, and maybe we could shift this particular discussion to the stream for more collective decision?